guides/Getting Started/API/index.md

```diff
@@ -1,53 +1,51 @@
+
+---
 ### `GET /applications` & `GET /groups` & `GET /messages`
 Retrieves information regarding applications, functional groups, or consumer friendly messages. An **id** (or additionally **uuid** for applications) can be specified so as to retrieve information for a specific item. Functional groups and consumer messages can be set to return templates containing all necessary information on that item being stored in the database. Applications can be filtered by approval status. If no parameters are specified `/applications` will return the latest version of each app, `/groups` and `/messages` will return the latest version of all functional groups or consumer messages in either production or staging mode.
 
+---
 ### `POST /applications/action`
 Updates an application's approval status. In the future this route will also notify the app's developer via email of the change in approval status.
 
+---
 ### `POST /applications/auto`
 If an application has been set to automatically approve all future updates then this route will validate the app uuid and update the approval status. In the future this route will also notify the app's developer via email of the change in approval status.
 
-### `POST /staging/policy` & `POST /production/policy`
-These are the routes sdl_core's default Policy Table should use when requesting a Policy Table update with either `/staging` or `/production` specified.
-Given a "shortened" Policy Table, the Policy Server will use that information to automatically construct a full Policy Table response and return it to the requester.
+---
+### `POST /applications/administrator`
+This route updates whether an app will have access to administrator functional groups.
 
-### `GET /policy/preview`
-This is the route hit by the Policy Server UI requesting a preview of the Policy Table. A variable **environment** indicates whether it is to be staging or production.
-
-### `POST /policy/apps`
-The Policy Server UI makes a request to this route which returns an example Policy Table segment for a particular app.
-
-### `POST /webhook`
-This is the route that should be specified on a company's page on the SDL Developer Portal (in the box titled Webhook URL under Company Info) to be hit by the SHAID server when an app has been updated.
-
-### `POST /permissions/update`
-This route queries SHAID for an update to the list of app permissions in the Policy Server.
-
-### `GET /permissions/unmapped`
-This route returns a list of permissions that are currently not attributed to any functional groups.
-
-### `POST /groups` & `POST /messages`
-These routes are hit by the Policy Server UI to update a functional group's/consumer message's information or to change its deleted status.
+---
+### `POST /applications/passthrough`
+This route update whether an app will be able to send unknown RPCs through App Service RPC Passsthrough.
```

Comment:
Inconsistent "App Service" name compared to "app services" name in the `PUT /applications/service/permission` route

Reply:
Updated names to all be "App Services"